apple/testing/default_runner/ios_test_runner.template.sh

```diff
@@ -267,3 +267,26 @@ if [[ -s "$error_file" || "$llvm_cov_status" -ne 0 ]]; then
   cat "$error_file" >&2
   exit 1
 fi
+
+llvm_cov_json_export_status=0
+if [[ "$TEST_ENV" =~ "COVERAGE_PRODUCE_JSON" ]]; then
+  mkdir -p "$TEST_UNDECLARED_OUTPUTS_DIR"
+  lcov_args=(
```

Comment:
we will have to mirror the args above about the specific binaries to correctly support hosted binaries. It looks to me like we can remove `-format lcov` from the `lcov_args` above, and pass that directly above, and then share all the rest of these arguments which will solve that and remove the need for this separate list?